src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -83,125 +73,103 @@ Session replay minimizes impact on your app's performance with:
+You can block entire sections of content on your site, which means the section will appear as an empty placeholder in session replay. For example, if you have an image on your About Us webpage and don’t need it captured, you could block the class containing the image.
 
-              ```html
-              <html>
-              <head>
-                <title>Sample image and text</title>
-              </head>
-              <body>
-                <div class="image-text-container nr-block">
-                  <img src="https://example.com/image.png" alt="Image description">
-                </div>
-              </body>
-              </html>
-              ```
-              
-            </Collapser>
-            </CollapserGroup>
+To block specific classes or attributes, you have a few options:
 
-* <DoNotTranslate>**Ignore**</DoNotTranslate> means user input will be ignored entirely. For example, if you want to ignore sensitive information like a user's email address and password, you could ignore the element that contains those fields.
+* On the Application settings page, add your CSS selectors in the **Block selectors* field.
 
+      <img
+          title="Block Selectors"
+          alt="Screenshot of session replay settings"
+          src={browserSessionReplayAppSettings}
+      />
+*  Manually add our CSS class `nr-block` or attribute `data-nr-block` to your webpage HTML. For example, if you wanted to block irrelevant text and images from session replay, add `nr-block` to `<div class>`:
 
+    ```html
+    <html>
+    <head>
+      <title>Sample image and text</title>
+    </head>
+    <body>
+      <div class="image-text-container nr-block">
+        <img src="https://example.com/image.png" alt="Image description">
+      </div>
+    </body>
+    </html>
+    ```
 
-        <CollapserGroup>
-        <Collapser
-        id="ignore-user-data"
-        title="How to ignore user data"
-    >
-        Add the CSS class `nr-ignore` to the input field’s class name. For example, if you want to ignore sensitive information on a billing page, add `class="nr-ignore"` to `<input type>`:
+### Ignore user input [#ignore-input]
 
-          ```html
-          <div class="sensitive-information">
-            <h2>Sensitive Information</h2>
-            <p>Credit card number: <input type="number" class="nr-ignore" id="creditCardNumber"></p>
-            <p>Expiration date: <input type="number" class="nr-ignore" id="expirationDate"></p>
-            <p>CVV code: <input type="number" class="nr-ignore" id="cvvCode"></p>
-          </div>
-          ```
-    </Collapser>
-    </CollapserGroup>
+You can ignore user input entirely, which means the input field won't show up at all in session replay. For example, if you want to ignore a user's email address and password, you could ignore the element that contains those fields.
```

Comment:
Ignoring means you ignore changes to the input field. The input field will still be displayed in the replay, but you won't see changes to the value.